lib/created_message_queue_handler.ts

```diff
@@ -243,11 +223,10 @@ export async function handleMessage(
     }
   }
 
-  // Try to get email notification metadata from user's profile
-  const maybeNotificationChannelEmail = tryGetEmailNotification(
-    maybeProfile,
-    defaultAddresses
-  );
+  // Try to get email notification metadata
```

Comment:
worth explaining what happens in both cases (email address exists in profile + email not defined)